extension/services/deezer/player.js

```diff
@@ -124,6 +140,39 @@ this.player = (function() {
         iconVolumeParent.click();
       }
     }
+
+    async playSection({ query, thenPlay, section }) {
+      try {
+        let SECTION;
+        await this.search(query);
+        try {
+          if (thenPlay) {
+            const anchorNodes = await this.waitForSelector(".container h2 a", {
+              all: true,
+              timeout: 5000,
+            });
+            for (const anchorTag of anchorNodes) {
+              if (anchorTag.innerText === section) {
+                SECTION = anchorTag.parentElement.parentElement;
```

Comment:
This should probably `break;` once it is found

Reply:
Oh. Sorry. Yes it was taking much time searching for all the sections thus making it slow.